x/superfluid/keeper/grpc_query.go

```diff
@@ -525,7 +530,7 @@ func (q Querier) TotalDelegationByValidatorForDenom(goCtx context.Context, req *
 			amount = amount.Add(record.DelegationAmount.Amount)
 		}
 
-		equivalentAmountOSMO, err := q.Keeper.GetSuperfluidOSMOTokensExcludeNative(ctx, req.Denom, amount)
+		equivalentAmountOSMO, err := q.Keeper.GetSuperfluidOSMOTokens(ctx, req.Denom, amount)
```

Comment:
Same question on this one.

Reply:
[abd9f3b](https://github.com/osmosis-labs/osmosis/pull/8320/commits/abd9f3b029aa40431e9431fb5bc60df9cd0e8b01)